text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,290 @@
+# Merge Elastic Common Schema with OpenTelemetry Semantic Conventions
+
+## Introduction
+
+This proposal is to merge the Elastic Common Schema (ECS) with the OpenTelemetry Semantic Conventions (SemConv) and provide full interoperability in OpenTelemetry component implementations. We propose to implement this by aligning the OpenTelemetry Semantic Conventions with [ECS FieldSets](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html#ecs-fieldsets) and vice versa where feasible. The long-term goal is to achieve convergence of ECS and OTel Semantic Conventions into a single open schema (e.g. OpenTelemetry Common Schema).
+
+## The Goal
+- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry. To kick off this effort, Elastic will nominate several domain experts to join the OpenTelemetry Semantic Convention Approvers. These approvers will be contributing, participating and co-maintaining the new standard.
```

Comment:
(I don’t foresee any issue, but removing this language makes it sounds less like a requirement, and this is already the definition of what OpenTelemetry Semantic Convention Approvers do) 
```suggestion
- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry. To kick off this effort, Elastic will nominate several domain experts to join the OpenTelemetry Semantic Convention Approvers.
```

Reply:
```suggestion
- Long-term, ECS and OTel SemConv will converge into one open standard that is maintained by OpenTelemetry. To kick off this effort, Elastic will nominate several domain experts to join the OpenTelemetry Semantic Convention Approvers to help with maintaining the new standard.
```